lnd.go

```diff
@@ -135,10 +135,20 @@ func lndMain() error {
 	signer := wc
 	bio := wc
 
+	lightningWalletConfig := &lnwallet.Config{
```

Comment:
As mentioned above, I see no reason as to why this new configuration parameter would need to be passed into the `lnwallet`.